selfdrive/car/hyundai/values.py

```diff
@@ -48,6 +48,7 @@ class HyundaiFlags(IntFlag):
   CANFD_HDA2 = 1
   CANFD_ALT_BUTTONS = 2
   CANFD_ALT_GEARS = 4
+  CANFD_CAMERA_SCC = 8
```

Comment:
Will think about unifying this in the future as we see more cars. Just didn't like how it wouldn't be true if CANFD cars were in that list based on HDA level, so splitting and using flags for now